src/schema.ts

```diff
@@ -0,0 +1,27 @@
+import { buildSchema } from 'graphql';
+
+export const typeDefs = buildSchema(`
+
+type Query {
+  hello: String
+}
+
+type Mutation {
+  createUser(data: UserInput): User
+}
+
+type User {
+  id: ID,
```

Comment:
```suggestion
  id: ID!
```

não tem vírgula, seria um ! ou vazio, de acordo com o fato da propriedade ser opcional ou não.
Inclusive recomendo dar uma checada em quais campos que são obrigatórios e marcar com o ! (diria que do UserInput só o birthDate seria opcional, por exemplo)

Reply:
Aqui no Apollo ele recomenda usar um comentário `#graphql`, que provavelmente vai ativar um syntax highlight no seu VSCode. Isso deve te ajudar a editar o schema e talvez pegar esse tipo de erro.

<img width="500" alt="image" src="https://github.com/indigotech/onboard-gabriel-moura/assets/6992731/a429ea41-dc0a-4308-ab6f-19687e9df129">